vars/gangplank.groovy

```diff
@@ -103,6 +103,60 @@ def runSpec(params =[:]) {
         gangplankCmd = _getMode(params)
         gangplankCmd += "--spec ${params['spec']} "
         runGangplank(gangplankCmd)
+    } else {
+        error("runSpec requires a spec param.")
+    }
+}
+
+// Available parameters:
+//     cmd:           string  -- the single command to run
+//     mode:          string  -- Gangplank Mode
+//     extraFlags:    string  -- Extra flags to use
+def runSingleCmd(params = [:]) {
+    if (params['cmd']) {
+        gangplankCmd = _getMode(params)
+        gangplankCmd += " --singleCmd \"${params['cmd']}\""
+        runGangplank(gangplankCmd)
+    } else {
+        error("runSingleCmd requires a cmd param.")
+    }
+}
+
+// A set of default parameters for building our FCOS AARCH64 builds via gangplank.
+// - use a locally build COSA because we don't currently build/push multi-arch COSA anywhere. 
+// - use pod mode with --podman for remote podman execution
+// - use the 'builds' "bucket" will fetch artifacts back into the expected 'builds' directory
+// - target aarch64 architecture
+def getGangPlankFCOSAARCH64Params() {
+    return [mode: 'pod',
+            extraFlags: '--podman --bucket=builds --arch=aarch64',
+            image: 'localhost/coreos-assembler:latest']
+}
```

Comment:
Why not fold this into `gangPlankFCOSAARCH64BuilderWrapper`? E.g. we check if `mode` is provided, otherwise default to `pod`, check for `image`, otherwise default to `localhost/coreos-assembler:latest`, etc... For flags, we can have an `appendFlags` or something if we want a pure append and not replacement. See this pattern in e.g. `cosaPod`.

Reply:
ok I'll see what I can do to merge this back in.

Reply:
moved functions to pipeline `gp.groovy`.